playbooks/roles/nghttpx/files/snakeoil.pem

```diff
@@ -0,0 +1,46 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
There shouldn't be a hardcoded private key used here. If you need a self-signed certificate generate it with a fresh key pair on the instance.